lib/arclight/traject/ead2_config.rb

```diff
@@ -234,12 +234,21 @@
   end
 
   to_field 'level_ssm' do |record, accumulator|
-    accumulator << record.attribute('level')
+    level = record.attribute('level')&.value
+    other_level = record.attribute('otherlevel')&.value
+
+    accumulator << if level == 'otherlevel'
+                     alternative_level = other_level if other_level
+                     alternative_level.present? ? alternative_level : 'Other'
```

Comment:
So it defaults to `Other` under when `"otherlevel"` is provided to the `@level` attribute, but `@otherlevel` is empty or non-existent?

Reply:
Yeah, that's how I read the existing indexing/testing code.